src/modules/handlers/info.ts

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line no-unused-vars
+import Koa from 'koa'
+import * as mongoose from 'mongoose'
+import { UserSchema } from '../../models/user'
+
+const User = mongoose.model('User', UserSchema)
+
+async function createUsers () {
+  await User.create({
+    name: 'Carolina Estrella',
+    email: 'carolina@host.com.br'
+  })
+}
+
+async function getUsers () {
+  return User.find()
+}
+
+async function info (ctx: Koa.Context) : Promise<void> {
+  await createUsers()
```

Comment:
Every time that info route receive a request you create that same data into database, this is how it suppose to work ?

Reply:
For now it is, but later I can create a post method to add data into the database.